css/ampdoc.css

```diff
@@ -276,6 +280,20 @@ amp-list[load-more] [load-more-end]
   display: none;
 }
 
+/**
+ * amp-list error messaging container should be hidden at first.
+ */
+ amp-list [fetch-error] {
```

Comment:
Saw these lines are also in `build-system/tasks/create-golden-css/css/main.css` -- does that need to be modified at all?

Reply:
Actually I think that golden file (last snapshot was ~2 years ago) shouldn't have been modified in the first place: https://github.com/ampproject/amphtml/blob/dd0fe4cca47573b9534cb19fbf86a573f1d0daf1/build-system/tasks/create-golden-css/css/main.css#L17-L22